src/main/java/appeng/me/storage/SecurityStationInventory.java

```diff
@@ -70,13 +69,13 @@ public IAEItemStack injectItems( final IAEItemStack input, final Actionable type
 		return input;
 	}
 
-	private boolean hasPermission( final BaseActionSource src )
+	private boolean hasPermission( final IActionSource src )
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Refactor this method to not always return the same value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3516)